README.md

```diff
@@ -1,24 +1,23 @@
-<a href="https://github.com/SydneyAstrophotonicInstrumentationLab/AMICAL">
-<img src="https://raw.githubusercontent.com/SydneyAstrophotonicInstrumentationLab/AMICAL/main/doc/Figures/amical_logo.png" width="300"></a>
+<a href="https://github.com/SAIL-Labs/AMICAL">
+<img src="https://raw.githubusercontent.com/SAIL-Labs/AMICAL/main/doc/Figures/amical_logo.png" width="300"></a>
 
 (**A**perture **M**asking **I**nterferometry **C**alibration and **A**nalysis
 **L**ibrary)
 
 [![PyPI](https://img.shields.io/pypi/v/amical.svg?logo=pypi&logoColor=white&label=PyPI)](https://pypi.org/project/amical/)
-[![PyPI](https://img.shields.io/badge/requires-Python%20≥%203.8-blue?logo=python&logoColor=white)](https://pypi.org/project/amical/)
```

Comment:
I removed this badge because it's kinda misleading (older releases support Python 3.7), and it's easy to forget to update when Python 3.8 is dropped too